src/Recordings.cpp

```diff
@@ -119,6 +177,11 @@ PVR_ERROR Recordings::GetRecordings(bool deleted, kodi::addon::PVRRecordingsResu
       }
     }
     m_iRecordingCount = recordingCount;
+    // force read disk space
+    m_checkedSpace = 0;
+    uint64_t total;
+    uint64_t used;
+    GetDriveSpace(total, used);
```

Comment:
Why have this here?

Reply:
> 
> Why have this here?

The check to the backend is every 5 minutes.  This recording list update is triggered right after a user does maintenance so by forcing the update, the user will get dynamic updates as shows are removed and timers scheduled to provide this information in a timely manner.

Reply:
Cool, makes sense.